stdlib/gc.mli

```diff
@@ -84,17 +84,21 @@ type stat =
 
 type control =
   { mutable minor_heap_size : int;
+    [@ocaml.deprecated_mutable "Use Gc.{(get()) with minor_heap_size = ...}"]
```

Comment:
I would personally recommend using `{(Gc.get()) with ...} `.  A local-open could lead to:
````ocaml
  let major = 42 in
  Gc.(set {(get ()) with major_heap_increment = major}) (* boom!  major is shadow by local open *)
````